cmd/power/power.go

```diff
@@ -114,23 +151,41 @@ func common(name string) {
 	}
 }
 
-func PowerOff() {
+func PowerOff(shutdown bool) {
 	common("poweroff")
-	reboot(syscall.LINUX_REBOOT_CMD_POWER_OFF)
+	if os.ExpandEnv("${IN_DOCKER}") == "true" {
+		PowerOff2(0, shutdown)
+	}
+}
+
+func PowerOff2(timeout int, shutdown bool) {
```

Comment:
We are not maintaining any API compatibility, we don't need to create "2" functions